src/graph/ShowExecutor.cpp

```diff
@@ -164,13 +164,19 @@ void ShowExecutor::showTags() {
         std::vector<std::string> header{"Name"};
         resp_->set_column_names(std::move(header));
 
+        std::set<std::string> uniqueTags;
```

Comment:
i think the order doesn't matter. Am I right?

Reply:
Users can get more friendly read when showing tags or edges